### PR TITLE
mingw-w64-thrift: added

### PR DESCRIPTION
**Pull request by emmenlau**

### CODE REVIEW

File: mingw-w64-thrift/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Maintainer: Mario Emmenlauer <memmenlauer@biodataanalysis.de>
+
+_realname=thrift
+pkgbase=mingw-w64-${_realname}-git
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+pkgver=1.0.0.3db41fa
+pkgrel=1
+pkgdesc="thrift - Apache Thrift interface definition language and binary communication protocol"
```
Comment (jeking3) [inline mingw-w64-thrift/PKGBUILD]: "Apache Thrift is a software framework for scalable cross-language services development." is what I use in the perl packaging.

File: mingw-w64-thrift/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Maintainer: Mario Emmenlauer <memmenlauer@biodataanalysis.de>
+
+_realname=thrift
+pkgbase=mingw-w64-${_realname}-git
```
Comment (jeking3) [inline mingw-w64-thrift/PKGBUILD]: Other packages I have seen for MinGW64 use the format w64-x86_64.  For example: "mingw-w64-x86_64-python2".

File: mingw-w64-thrift/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+pkgver=1.0.0.3db41fa
```
Comment (jeking3) [inline mingw-w64-thrift/PKGBUILD]: Any way we can derive this from the latest release branch of thrift, for example 0.10.0, and use the github 0.10.0 tag to do the build?

File: mingw-w64-thrift/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+         "${MINGW_PACKAGE_PREFIX}-openssl"
+         "${MINGW_PACKAGE_PREFIX}-zlib")
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
+             "${MINGW_PACKAGE_PREFIX}-cmake"
+             "make"
+             "bison"
+             "flex"
+             "git")
+options=('strip')
+source=(git+https://github.com/apache/thrift.git#commit=3db41fa)
```
Comment (jeking3) [inline mingw-w64-thrift/PKGBUILD]: Can we use a tag like 0.10.0 instead?

File: mingw-w64-thrift/PKGBUILD (hunk 2 above)
Comment (mati865) [inline mingw-w64-thrift/PKGBUILD]: @jeking3 `pkgbase` is fine. You must have mistaken it for `pkgname` which have `MINGW_PACKAGE_PREFIX` variable inside expanding to `mingw-w64-i686` or `mingw-w64-x86_64`.

**Comment by emmenlau** [comment]
Hi @jeking3 , I reported this new package here https://issues.apache.org/jira/browse/THRIFT-4159
upstream for the thrift maintainers. Can you elaborate what you mean with "It would be nice to add
this to the thrift project itself"? It should be as easy and good for you as possible, let me know what
I can do to improve?

File: mingw-w64-thrift/PKGBUILD (hunk 3 above)
Comment (emmenlau) [inline mingw-w64-thrift/PKGBUILD]: Yes I can modify it once you have a new tag. Currently I need the commit `3db41fa` to have
a successful build, so we need to wait for the next tag before I can switch. Is that good?

File: mingw-w64-thrift/PKGBUILD (hunk 1 above)
Comment (emmenlau) [inline mingw-w64-thrift/PKGBUILD]: Done, updated in latest PR.

File: mingw-w64-thrift/PKGBUILD (hunk 4 above)
Comment (emmenlau) [inline mingw-w64-thrift/PKGBUILD]: See my comment above: yes, as soon as commit `3db41fa` is in a tag?

File: mingw-w64-thrift/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# Maintainer: Mario Emmenlauer <memmenlauer@biodataanalysis.de>
+
+_realname=thrift
+pkgbase=mingw-w64-${_realname}-git
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+pkgver=1.0.0.3db41fa
+pkgrel=1
+pkgdesc="thrift - Apache Thrift is a software framework for scalable cross-language services development"
```
Comment (mati865) [inline mingw-w64-thrift/PKGBUILD]: `(mingw-w64)` text is missing.

Comment (emmenlau) [inline mingw-w64-thrift/PKGBUILD]: Thanks @mati865 , fixed!

**Pull request merged**